vcs/git/git.go

```diff
@@ -352,23 +352,18 @@ func gitCMD(args ...string) (out string, err error) {
 	cmd := exec.Command("git", args...)
 	b, err := cmd.CombinedOutput()
 	out = string(b)
-	return out, errors.Annotate(err, out)
 
-	// TODO(waigani) stdout is empty?
-	// cmd := exec.Command("git", args...)
-	// e := &bytes.Buffer{}
-	// o := &bytes.Buffer{}
-	// cmd.Stderr = e
-	// cmd.Stdout = o
-	// stderr = string(e.Bytes())
-	// stdout = string(o.Bytes())
-	// err = cmd.Run()
-	// if err != nil {
-	// 	gitargs := strings.Join(args, " ")
-	// 	return "", stderr, errors.Annotate(err, "git args: `"+gitargs+"` stdout: "+stdout+" stderr: "+stderr)
-
-	// }
-	// return strings.TrimSpace(stdout), stderr, nil
+	if isMatch, _ := regexp.MatchString("unknown switch `4'", out); isMatch {
```

Comment:
It's better to check the value or `err` than stdout: we may change the formatting of stdout at some point, but the underlying `err` will remain the same.

regexp is also quite expensive and verbose. Try the strings package:

```go
	if strings.Contains(err.Error(), "unknown switch `4'") {
		// rerun
	}
```